extension/src/background/messageListener/popupMessageListener.ts

```diff
@@ -152,12 +158,15 @@ interface KeyPair {
 
 export const popupMessageListener = (request: Request, sessionStore: Store) => {
   const localStore = dataStorageAccess(browserLocalStorage);
-  const localKeyStore = new KeyManagerPlugins.BrowserStorageKeyStore();
-  localKeyStore.configure({ storage: browserLocalStorage });
+  const localKeyStore = new BrowserStorageKeyStore();
+  // ts-wallet-sdk storage area definition clashes with webkit polyfills
```

Comment:
These types are incorrect in ts-wallet-sdk, this PR should fix that https://github.com/stellar/typescript-wallet-sdk/pull/129